babadee001/HelloBooks
# #154357305 Remove redundant files

#### What does this PR do?
Remove redundant files
#### Description of Task to be completed?
- Remove all console log statements
- Reference data appropriately using environment variables
#### How should this be manually tested?
- Open the repo
- Check through the code
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#154357305
#### Screenshots (if appropriate)
None
#### Questions:
None
